.scrutinizer.yml

```diff
@@ -1,5 +1,7 @@
 build:
     environment:
+        variables:
+            COLUMNS: 120
```

Comment:
We're not running the tests on scrutinizer so I don't think this is needed

Reply:
I will try, but before, Scrutinizer was failing

Reply:
Looks, like tests were executed by Scrutinizer:
https://scrutinizer-ci.com/g/doctrine/doctrine2/inspections/fe506200-ebd0-465d-b9a1-1f4b7515fd8a

Reply:
Okay, I'll fix scrutinizer config, but it's failing because it's using the default phpunit config (`phpunit.xml.dist`) that hasn't been updated.

Reply:
@OskarStark #6843 fixes the scrutinizer configuration